integration/run_test.go

```diff
@@ -448,22 +430,19 @@ func TestKritisPods(t *testing.T) {
 		},
 	}
 
-	// CRDs themselves are non-namespaced so we have to delete them each run
-	deleteCRDExamples()
-
-	deleteKritis := initKritis(t)
+	ns, deleteNs := setupNamespace(t)
+	defer deleteNs()
+	createGACSecret(t, ns)
+	deleteKritis := initKritis(t, ns)
 	defer deleteKritis()
-	if err := kubernetesutil.WaitForDeploymentToStabilize(client, "default",
+	if err := kubernetesutil.WaitForDeploymentToStabilize(client, ns.Name,
 		"kritis-validation-hook", 2*time.Minute); err != nil {
 		t.Fatalf("Timed out waiting for deployment to stabilize")
 	}
 	createCRDExamples(t)
-	time.Sleep(5 * time.Second)
+	time.Sleep(10 * time.Second)
```

Comment:
Why double the sleep? Is it possible to replace the sleep statement with something that polls for when the time is appropriate to proceed?

Reply:
For sure, I ended up moving createCRDExamples() before the check for the stabilized deployment which should (hopefully) be enough time

Reply:
You should be able to check if the ISP is up by looking waiting for a successful return code for 'kubectl get imagesecuritypolicy my-isp'.  The stabilized deployment check won't really help verify that the ISP is created

Reply:
oh perfect, i'll add that in, thanks!